templates/blog/post_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+{%  block content %}
+    <div class="index">
+    <h1>Post List</h1>
+    {% for posts in page_obj %}
```

Comment:
The loop variable is named `posts`, which is plural. It's more conventional to use a singular name like `post` when iterating over a collection of items. Consider renaming it to `post` for clarity.